p2p/protocol/identify/pb/identify.proto

```diff
@@ -36,4 +36,11 @@ message Identify {
 
   // a delta update is incompatible with everything else. If this field is included, none of the others can appear.
   optional Delta delta = 7;
+
+  // signedPeerRecord contains a serialized SignedEnvelope containing a PeerRecord,
+  // signed by the sending node. It contains the same addresses as the listenAddrs field, but
+  // in a form that lets us share authenticated addrs with other peers.
+  // see github.com/libp2p/go-libp2p-core/record/pb/envelope.proto and
+  // github.com/libp2p/go-libp2p-core/peer/pb/peer_record.proto for message definitions.
+  optional bytes signedPeerRecord = 8;
```

Comment:
Can we make this repeated? That way, we can have one record per _scope_ (and we can add scope info to the signed peer record itself).

Reply:
Note: we can also do this in a new protocol version.

Reply:
@Stebalien 

If we do this in a new protocol version, will we make a "breaking" protocol change for it or add a new field for the multiple scoped records ?

Reply:
I'd make a breaking protocol change.